packages/arb-token-bridge-ui/src/pages/api/cctp/[type].ts

```diff
@@ -138,6 +138,18 @@ export default async function handler(
       return
     }
 
+    // if invalid pageSize, send empty data instead of error
```

Comment:
The invalid pageSize is just a fail-safe, since API's can be called from anywhere, it's good to limit here. 
The original issue was pageSize===0 being passed by transaction history code during it's initialisation, which was fixed by this check.